client.go

```diff
@@ -297,8 +297,9 @@ func NewClient(cfg *ClientConfig) (cl *Client, err error) {
 			}
 			return t.announceRequest(event), nil
 		},
-		Proxy:       cl.config.HTTPProxy,
-		DialContext: cl.config.TrackerDialContext,
+		Proxy:                       cl.config.HTTPProxy,
+		WebtorrentTrackerHttpHeader: cl.config.WebtorrentTrackerHttpHeader,
```

Comment:
What made you choose Webtorrent over Websocket? I guess I named it webtorrent in the repo, but websocket without the tracker context isn't very specific. Just wonder if you have any thoughts either way.

Reply:
I was a bit unsure about which to choose, to be honest. I think Webtorrent might make it easier to understand what the purpose of setting this header is (communicating with a webtorrent tracker) when defining the client config. Also, I thought it would be consistent with the way the project is organised, with all websocket related parts collected under webtorrent (as far as I can tell). 
However, I realise that it might be more consistent if a type named `websocketTrackers` had a field called `WebsocketTrackerHttpHeader`... Or we might simply name it `HttpHeader` in `websocketTrackers` and something more descriptive in `Client.config`.

All in all, I don't really have a strong preference either way, whatever will make it easier to understand what the function is supposed to do :)